e2e-tests/portal/cypress/e2e/journeys/payment-report-officer/record-corrections/feature-flag-enabled/correction-sent.spec.js

```diff
@@ -81,10 +81,10 @@ context('Correction sent page - Fee record correction feature flag enabled', ()
       cy.assertText(correctionSent.ukefNotifiedText(), 'UKEF will be notified via email that a record has been updated.');
     });
 
-    it('should redirect to the Report GEF utilisation and fees paid when continue is clicked', () => {
+    it('should redirect to the report upload page when continue is clicked', () => {
       cy.clickContinueButton();
 
-      cy.assertText(mainHeading(), 'Report GEF utilisation and fees');
+      cy.url().should('eq', relative('/utilisation-report-upload'));
```

Comment:
The title of this page depends on whether a report is due for upload which is covered by other tests so to keep this less brittle I have updated it to check the url which is always the same